db/migrate/20160702235909_add_column_to_business.rb

```diff
@@ -0,0 +1,5 @@
+class AddColumnToBusiness < ActiveRecord::Migration
+  def change
+    add_column :businesses, :description, :string
```

Comment:
Excellent. We needed this. I'll have to change the business show page to display the description. Do we want to display it on the business index page also?